File: test/fixtures/vuln-crypto.js
const crypto = require("crypto");
const cipher = crypto.createCipheriv("aes-128-cbc", key, "hardcodediv");
const rsa = crypto.generateKeyPairSync("rsa", { modulusLength: 1024 });
const token = crypto.random(16);
const salt = "hardcodedsalt456";

File: test/fixtures/vuln-crypto.py
from Crypto.Cipher import AES
from Crypto.PublicKey import RSA
import hashlib

cipher = AES.new(key, AES.MODE_ECB)
iv = b"hardcodediv12345"
aes = AES.new(key, AES.MODE_CBC, iv)
rsa_key = RSA.generate(1024)
salt = b"hardcodedsalt123"
h = hashlib.md5(password.encode()).hexdigest()
